### ISSUE TITLE
AccordionPanel missing aria-level API

### ISSUE THREAD

**Issue by knoobie** [vaadin/flow-components#9741]
### Description

The webcomponent says: 

>  * The WAI ARIA Best Practices also recommend setting `aria-level` depending
>   * on what level the headings are. It is hard to determine the level of a heading
>   * algorithmically, and setting it is not strictly required to have an accessible
>   * accordion. To keep things easier to use, this element does not set `aria-level`
>   * attribute but leaves that to the developer creating an accordion.

### Expected outcome

Accordion::setAriaLevel is present and automatically applies the aria-level to all AccordionPanel 

AccordionPanel:.setAriaLevel is present and applies aria-label to the corresponding panel

### Minimal reproducible example

.

### Steps to reproduce

.

### Environment

Vaadin version(s): latest

**Comment by web-padawan** [comment]
Thanks for the issue. This is a duplicate of https://github.com/vaadin/web-components/issues/6900.
